modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -240,13 +240,14 @@ than 48 characters in length.
 ====
 {micro} is designed to update itself automatically by default and will reboot after applying updates.
 However, this behavior is not desirable for the {productname} environment.
-To prevent automatic updates on your server, we recommend disabling the transactional-update timer.
+To prevent automatic updates on your server, {productname} disables the
+transactional-update timer during the bootstrap process.
 
-You can disable the timer by running the following command:
+If you prefer to keep the behavior, enable the timer by running the following command:
```

Comment:
```suggestion
If you prefer the {micro} default behavior, enable the timer by running the following command:
```
I think we'd better still assume that the user works with GMC for the first time with suma on Micro.